core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -74,8 +74,7 @@ case class LogReadResult(info: FetchDataInfo,
   }
 
   override def toString =
-    s"Fetch Data: [$info], HW: [$hw], leaderLogEndOffset: [$leaderLogEndOffset], readSize: [$readSize], error: [$error]"
-
+    s"Fetch Data: [$info], HW: [$hw], leaderLogEndOffset: [$leaderLogEndOffset], readSize: [$readSize], error: [${error.exceptionName}]"
```

Comment:
Sure, I agree we are not consistent across the board. So I'll leave that out in the next update.